meta-resin-common/recipes-connectivity/networkmanager/files/NetworkManager.conf

```diff
@@ -3,4 +3,4 @@ dns=default
 plugins=keyfile
 
 [keyfile]
-unmanaged-devices=interface-name:docker*;tun*
+unmanaged-devices=interface-name:docker*;interface-name:tun*;interface-name:resin*;interface-name:veth*;
```

Comment:
Also, do we need to ignore the vpn interface here? Isn't that created after NM starts so therefore not affected?